### ISSUE TITLE
Error while creating BillModel

### ISSUE THREAD

**Issue by KipCrossing** [voteflux/voting_app#156]
For the current master version:

```bash
I/flutter (12839): Bill <id: r6409; shortTitle: Australian Citizenship Amendment (Citizenship Cessation) 2020> | 0 | 0
I/flutter (12839): Error while creating BillModel
I/flutter (12839): Stack trace:
I/flutter (12839):  #0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:46:39)
I/flutter (12839): #1      _AssertionError._throwNew (dart:core-patch/errors_patch.dart:36:5)
I/flutter (12839): #2      _GetItImplementation._findFactoryByNameAndType (package:get_it/get_it_impl.dart:298:9)
I/flutter (12839): #3      _GetItImplementation.get (package:get_it/get_it_impl.dart:313:29)
I/flutter (12839): #4      _GetItImplementation.call (package:get_it/get_it_impl.dart:339:12)
I/flutter (12839): #5      $initGetIt.<anonymous closure> (package:voting_app/locator.config.dart:44:12)
I/flutter (12839): #6      _ServiceFactory.getObject (package:get_it/get_it_impl.dart:127:36)
I/flutter (12839): #7      _GetItImplementation.get (package:get_it/get_it_impl.dart:326:34)
I/flutter (12839): #8      _GetItImplementation.call (package:get_it/get_it_impl.dart:339:12)
I/flutter (12839): #9      new _BillListItemState (package:voting_app/ui/widgets/bill_list_item.dart:32:32)
I/flutter (12839): #10     BillListItem.createState (package:voting_app/ui/widgets/bill_list_item.dart:14:39)
I/flutter (12839): #11     new StatefulElement (package:flutter/src/widgets/framework.dart:4716:25)
I/flutter (12839): #12     StatefulWidget.createElement (pac
```

**Comment by KipCrossing** [comment]
More

```bash
════════ Exception caught by widgets library ═══════════════════════════════════════════════════════
The following assertion was thrown building NotificationListener<KeepAliveNotification>:
Object/factory with  type Bill is not registered inside GetIt. 
(Did you accidentally do  GetIt sl=GetIt.instance(); instead of GetIt sl=GetIt.instance;
Did you forget to register it?)
'package:get_it/get_it_impl.dart':
Failed assertion: line 298 pos 9: 'instanceFactory != null'

The relevant error-causing widget was: 
  SliverList-[List<Bill>#382c1] file:///home/kipling/Programs/voting_app/lib/ui/views/all_bills_view.dart:52:23
When the exception was thrown, this was the stack: 
#2      _GetItImplementation._findFactoryByNameAndType (package:get_it/get_it_impl.dart:298:9)
#3      _GetItImplementation.get (package:get_it/get_it_impl.dart:313:29)
#4      _GetItImplementation.call (package:get_it/get_it_impl.dart:339:12)
#5      $initGetIt.<anonymous closure> (package:voting_app/locator.config.dart:44:12)
#6      _ServiceFactory.getObject (package:get_it/get_it_impl.dart:127:36)
...
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by widgets library ═══════════════════════════════════════════════════════
'package:flutter/src/rendering/sliver_multi_box_adaptor.dart': Failed assertion: line 259 pos 16: 'child == null || indexOf(child) > index': is not true.
The relevant error-causing widget was: 
  SliverList-[List<Bill>#382c1] file:///home/kipling/Programs/voting_app/lib/ui/views/all_bills_view.dart:52:23
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by widgets library ═══════════════════════════════════════════════════════
'package:flutter/src/rendering/sliver_multi_box_adaptor.dart': Failed assertion: line 259 pos 16: 'child == null || indexOf(child) > index': is not true.
The relevant error-causing widget was: 
  SliverList-[List<Bill>#382c1] file:///home/kipling/Programs/voting_app/lib/ui/views/all_bills_view.dart:52:23
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by widgets library ═══════════════════════════════════════════════════════
'package:flutter/src/rendering/sliver_multi_box_adaptor.dart': Failed assertion: line 259 pos 16: 'child == null || indexOf(child) > index': is not true.
The relevant error-causing widget was: 
  SliverList-[List<Bill>#382c1] file:///home/kipling/Programs/voting_app/lib/ui/views/all_bills_view.dart:52:23
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by rendering library ═════════════════════════════════════════════════════
'package:flutter/src/widgets/framework.dart': Failed assertion: line 2687 pos 20: '_debugCurrentBuildTarget == context': is not true.
The relevant error-causing widget was: 
  SliverList-[List<Bill>#382c1] file:///home/kipling/Programs/voting_app/lib/ui/views/all_bills_view.dart:52:23
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by rendering library ═════════════════════════════════════════════════════
Null check operator used on a null value
The relevant error-causing widget was: 
  CustomScrollView file:///home/kipling/Programs/voting_app/lib/ui/views/all_bills_view.dart:30:19
════════════════════════════════════════════════════════════════════════════════════════════════════
```

**Comment by KipCrossing** [comment]
Fixed in #160
